docs/user/auth-resources.md

```diff
@@ -0,0 +1,56 @@
+|`GET /api/v1/devices/{name}/rendered`|`GetRenderedDeviceSpec`|`devices`|`getrenderedspec`|
+|`PUT /api/v1/devices/{name}/decommission`|`DecommissionDevice`|`devices`|`decomission`|
+|`POST /api/v1/enrollmentrequests`|`CreateEnrollmentRequest`|`enrollmentrequests`|`create`|
+|`GET /api/v1/enrollmentrequests`|`ListEnrollmentRequests`|`enrollmentrequests`|`list`|
+|`DELETE /api/v1/enrollmentrequests`|`DeleteEnrollmentRequests`|`enrollmentrequests`|`deletecollection`|
+|`GET /api/v1/enrollmentrequests/{name}`|`ReadEnrollmentRequest`|`enrollmentrequests`|`read`|
+|`PUT /api/v1/enrollmentrequests/{name}`|`ReplaceEnrollmentRequest`|`enrollmentrequests`|`replace`|
+|`DELETE /api/v1/enrollmentrequests/{name}`|`DeleteEnrollmentRequest`|`enrollmentrequests`|`delete`|
+|`GET /api/v1/enrollmentrequests/{name}/status`|`ReadEnrollmentRequestStatus`|`enrollmentrequests`|`readstatus`|
+|`POST /api/v1/enrollmentrequests/{name}/approval`|`ApproveEnrollmentRequest`|`enrollmentrequests`|`approve`|
+|`PUT /api/v1/enrollmentrequests/{name}/status`|`ReplaceEnrollmentRequestStatus`|`enrollmentrequests`|`replacestatus`|
+|`POST /api/v1/fleets`|`CreateFleet`|`fleets`|`create`|
+|`GET /api/v1/fleets`|`ListFleets`|`fleets`|`list`|
+|`DELETE /api/v1/fleets`|`DeleteFleets`|`fleets`|`deletecollection`|
+|`GET /api/v1/fleets/{name}`|`ReadFleet`|`fleets`|`read`|
+|`PUT /api/v1/fleets/{name}`|`ReplaceFleet`|`fleets`|`replace`|
+|`DELETE /api/v1/fleets/{name}`|`DeleteFleet`|`fleets`|`delete`|
+|`GET /api/v1/fleets/{name}/status`|`ReadFleetStatus`|`fleets`|`readstatus`|
+|`PUT /api/v1/fleets/{name}/status`|`ReplaceFleetStatus`|`fleets`|`replacestatus`|
+|`POST /api/v1/repositories`|`CreateRepository`|`repositories`|`create`|
+|`GET /api/v1/repositories`|`ListRepositories`|`repositories`|`list`|
+|`DELETE /api/v1/repositories`|`DeleteRepositories`|`repositories`|`deletecollection`|
+|`PUT /api/v1/repositories/{name}`|`ReplaceRepository`|`repositories`|`replace`|
+|`DELETE /api/v1/repositories/{name}`|`DeleteRepository`|`repositories`|`delete`|
+|`POST /api/v1/resourcesyncs`|`CreateResourceSync`|`resourcesyncs`|`create`|
+|`GET /api/v1/resourcesyncs`|`ListResourceSync`|`resourcesyncs`|`list`|
+|`DELETE /api/v1/resourcesyncs`|`DeleteResourceSyncs`|`resourcesyncs`|`deletecollection`|
+|`GET /api/v1/resourcesyncs/{name}`|`ReadResourceSync`|`resourcesyncs`|`read`|
+|`PUT /api/v1/resourcesyncs/{name}`|`ReplaceResourceSync`|`resourcesyncs`|`replace`|
+|`DELETE /api/v1/resourcesyncs/{name}`|`DeleteResourceSync`|`resourcesyncs`|`delete`|
+|`GET /api/v1/api/v1/fleets/{fleet}/templateVersions`|`ListTemplateVersions`|`templateversions`|`list`|
+|`DELETE /api/v1/api/v1/fleets/{fleet}/templateVersions`|`DeleteTemplateVersions`|`templateversions`|`deletecollection`|
+|`GET /api/v1/fleets/{fleet}/templateVersions/{name}`|`ReadTemplateVersion`|`templateversions`|`read`|
+|`DELETE /api/v1/fleets/{fleet}/templateVersions/{name}`|`DeleteTemplateVersion`|`templateversions`|`delete`|
```

Comment:
This needs some thinking, as templateVersions is a subresource of a specific fleet.
I think the resource should be called `fleets/templateVersions` and the verbs list/deletecollection and get/delete.